BLSQ/iaso
# POLIO-1662 Supply Chain: VRF & DG Approval include options of 'missing' and 'not required'

Supply Chain: VRF & DG Approval include options of 'missing' and 'not required'

Related JIRA tickets : POLIO-1662

## Self proofreading checklist


## Changes

- Add a “type” on the VRF model with default “Normal”, with other options “Missing” and “Not Required”.
- When its Not Required, the optional fields of the VRF will be hidden, as well as the the Pre Alerts and VAR tabs.
- Field Quantity Ordered In Doses must become optional (Default to 0)

## How to test

See video below, but basically go to Vaccine Supply Chain and create a new entry or edit an existing entry and try to play with the VRF Type field

## Print screen / video


https://github.com/user-attachments/assets/6bdfa0d1-9d5d-41bd-ba69-178a21e34859
